package.json

```diff
@@ -28,7 +28,8 @@
 		"match-sorter": "6.3.1",
 		"papaparse": "5.4.1",
 		"serialize-error": "11.0.1",
-		"strife.js": "1.0.6"
+		"strife.js": "^1.0.6",
+		"undici": "^5.23.0"
```

Comment:
why did you instal undici

Reply:
after i fixed strife.js undici broke so i uninstalled and reinstalled it and it worked, i didnt change package.json manually

Reply:
well this will need to be reverted before merge

Reply:
`npm rm undici`